### ISSUE TITLE
Plans?

### ISSUE THREAD

**Issue by IanVS** [Intellicode/eslint-plugin-redux#1]
Hi, it looks like this package has existed for 4 months without any development.  Are you still planning to create an ESLint plugin for redux?  If not, it might be nice of you to stop sitting on the npm package name so that someone else could create one.

**Comment by Intellicode** [comment]
Hi @IanVS I definitely have plans, but due to circumstances and also my other Eslint plugin I haven't pushed any code yet. Lets say that if I don't have anything in a month, someone else can take it over if he/she wants. How does that sound?

**Comment by IanVS** [comment]
Yeah no problems, I know how it goes.  I was just curious more than anything.  I'm excited to see what's in store.  Perhaps you could start by at least adding some info to the readme with some ideas for rules that you plan to make, so contributors can help if they feel so inclined.

**Comment by Intellicode** [comment]
@IanVS I'll definitely do that, perhaps there will be interested to get the discussion started on my initial idea!

**Comment by DianaSuvorova** [comment]
I think generic redux eslint rules would be rather difficult to articulate because of [its (redux's) simplicity](https://github.com/reactjs/redux#the-gist). It is more practical to enforce specific middleware/connector best practices. For example: [saga](https://github.com/pke/eslint-plugin-redux-saga)  or [react-redux](https://github.com/DianaSuvorova/eslint-plugin-react-redux) (<- self ad).

**Comment by Intellicode** [comment]
@DianaSuvorova I think so too, hence the lack of activity so far :)
